finagle-core/src/main/scala/com/twitter/finagle/util/LoadService.scala

```diff
@@ -21,10 +21,14 @@ import scala.io.Source
  */
 private object ClassPath {
 
-  private val ignoredPackages = Seq(
+  private val defaultIgnoredPackages = Seq(
     "apple/", "ch/epfl/", "com/apple/", "com/oracle/",
     "com/sun/", "java/", "javax/", "scala/", "sun/", "sunw/")
 
+  private[util] def ignoredPackages =
```

Comment:
This method was done as suggested along with adding of the val for the default ignored packages. We are not sure how much of a performance hit it would be to calculate this as a def on each call but this would definitely ensure no regressions as well as avoid any explicit method calls etc to update them.